workers/ohsome_quality_analyst/base/indicator.py

```diff
@@ -63,19 +63,19 @@ class BaseIndicator(metaclass=ABCMeta):
     def __init__(
         self,
         layer_name: str,
-        bpolys: FeatureCollection = None,
-        data: dict = None,
+        feature: Feature,
```

Comment:
I don't think this is true. In the `api.py` you check if `feature` is a `Feature` or a `Geometry`.

```suggestion
        feature: Union[Feature, Geometry],
```
(don't forget the import)

Reply:
See https://github.com/GIScience/ohsome-quality-analyst/pull/69#discussion_r664022120